test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,2368 @@
+          this.enableAnalytics({ endpoint });
+
+          sandbox.stub(navigator, 'sendBeacon').returns(true);
+
+          performStandardAuction();
+          sandbox.clock.tick(this.defaultTimeout + 1);
+
+          assert.calledOnce(navigator.sendBeacon);
+          assert.calledWith(navigator.sendBeacon, endpoint, sinon.match.string);
+          assert.calledWith(log.info, `Analytics report sent to ${endpoint}`, sinon.match.object);
+
+          navigator.sendBeacon.restore();
+        });
+      });
+    });
+
+    context('when an error occurs while sending the AnalyticsReport', function() {
+      it('logs an error', function() {
+        this.enableAnalytics();
+
+        sandbox.stub(navigator, 'sendBeacon').returns(false);
+
+        performStandardAuction();
+        sandbox.clock.tick(this.defaultTimeout + 1);
+
+        assert.calledOnce(navigator.sendBeacon);
+        assert.calledOnce(log.error);
+        assert.calledWith(log.error, 'Analytics report exceeded User-Agent data limits and was not sent.', sinon.match.object);
+
+        navigator.sendBeacon.restore();
+      });
+    });
+
+    context('when a BID_WON event is received', function() {
+      context('and there is no record of that bid being requested', function() {
+        it('logs a warning message', function() {
+          this.enableAnalytics();
+
+          const mockEvents = getMockEvents();
+          const { prebid } = mockEvents;
+          const [ auction ] = prebid;
+
+          events.emit(EVENTS.AUCTION_INIT, auction.AUCTION_INIT);
+
+          const fakeBidWonEvent = Object.assign(auction.BID_WON[0], {
+            transactionId: 'foo'
+          })
+
+          events.emit(EVENTS.BID_WON, fakeBidWonEvent);
+
+          assert.calledWithExactly(log.warn, 'transactionId "foo" was not found. Nothing to enqueue.');
+        });
+      });
+    });
+
+    context('when the same BID_REQUESTED event is received more than once', function() {
+      it('logs a warning message', function() {
+        this.enableAnalytics();
+
+        const mockEvents = getMockEvents();
+        const { prebid } = mockEvents;
+        const [ auction ] = prebid;
+
+        events.emit(EVENTS.AUCTION_INIT, auction.AUCTION_INIT);
+
+        events.emit(EVENTS.BID_REQUESTED, auction.BID_REQUESTED[0]);
+        events.emit(EVENTS.BID_REQUESTED, auction.BID_REQUESTED[0]);
+
+        assert.calledWithExactly(log.warn, 'transactionId "ef947609-7b55-4420-8407-599760d0e373" already exists')
+      });
+    });
+
+    context('when a transaction does not reach its complete state', function() {
+      context('and a timeout config value has been given', function() {
+        context('and the timeout value has elapsed', function() {
+          it('logs a warning', function() {
+            this.enableAnalytics({ timeout: 2000 });
+
+            performAuctionWithMissingBidWon();
+
+            sandbox.clock.tick(5001);
+
+            assert.called(log.warn);
```

Comment:
there's no need for this assertion.. the one below covers the call.

Reply:
Agreed, this should be removed.